### ISSUE TITLE
Conflict on multiplication

### ISSUE THREAD

**Issue by Tarlan0001** [chakravala/Reduce.jl#17]
```
julia> using ForceImport
julia> @force using Reduce.Algebra
Reduce (Free CSL version, revision 4534), 05-Apr-18 ...
WARNING: ignoring conflicting import of Algebra.* into Main

julia> a=:x
:x

julia> a*1
ERROR: MethodError: no method matching *(::Symbol, ::Int64)
Closest candidates are:
  *(::Any, ::Any, ::Any, ::Any...) at operators.jl:424
  *(::Bool, ::T<:Number) where T<:Number at bool.jl:101
  *(::Complex{Bool}, ::Real) at complex.jl:255
```

**Comment by Tarlan0001** [comment]
I did it after restart of the Terminal, still ignores `*`

**Comment by Tarlan0001** [comment]
Worked after calling `workspace()` in the beginning. But it's too long

**Comment by chakravala** [comment]
Are you using Windows? Have you checked out the master branch or are you on a release version? I believe this problem only occurs for windows users on Julia v0.6, it is fixed on Julia 0.7. There is no workaround for this, aside from doing something like `* = Algebra.:*`, unfortunately, due to an issue with how Julia 0.6 assigns the `*` operator only with WIndows on Julia 0.6 (not on julia 0.7).
